action_server/src/robocorp/action_server/_actions_import.py

```diff
@@ -120,6 +120,10 @@ def import_action_package(
     if not name:
         name = import_path.name
 
+    log.info(
+        f"Python Interpretor path: {use_env.get('RCC_HOLOTREE_SPACE_ROOT')}/bin/python"
```

Comment:
This only works for linux and mac.
cross-platform way would be to state something like:
`f"Action Package Python environment at: {use_env.get('RCC_HOLOTREE_SPACE_ROOT')}"`